libs/design/src/atoms/form/checkbox/checkbox.component.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  Component,
+  Input,
+  HostBinding,
+  ChangeDetectionStrategy,
+  EventEmitter,
+  Output,
+  ChangeDetectorRef,
+  ViewChild,
+  ElementRef
+} from '@angular/core';
+
+let checkboxIdNum = 0;
+
+@Component({
+  selector: 'daff-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+
+})
+export class DaffCheckboxComponent {
+  @ViewChild('inputElement', {static: true, read: ElementRef}) nativeCheckbox: ElementRef<HTMLInputElement>;
+  /**
+   * The name of the checkbox.
+   */
+  @Input() name: string;
+  /**
+   * The value of the checkbox.
+   */
+  @Input() value: any;
+  /**
+   * Boolean value to determine whether or not the checkbox is checked.
+   */
+  _checked = false;
+  @Input()
+  get checked() {
+    return this._checked;
+  }
+  set checked(value: boolean) {
+    if (this._checked === value) {
+      return;
+    }
+    if (value === true) {
+      this.nativeCheckbox.nativeElement.checked = true;
+      this.becameChecked.emit(this._checked);
+    }
+    else {
+      this.nativeCheckbox.nativeElement.checked = false;
+      this.becameUnchecked.emit();
+    }
+
+    this._checked = value;
+  }
+  /**
+   * The id of the checkbox. Must be unique. If not entered by a user then it is generated.
+   */
+  @Input() id: string = 'daff-checkbox-' + checkboxIdNum;
+  /**
+   * The aria-label of the checkbox. If not set by user then it defaults to the name of the checkbox.
+   */
+  @Input() label = name;
+  /**
+   * The aria-labeledby of the checkbox.
+   */
+  @Input() labeledBy: string;
```

Comment:
Can this be computed from `ariaLabeledBy` on the host as opposed to a new input?